.github/workflows/rust.yml

```diff
@@ -34,6 +34,16 @@ jobs:
     - uses: dtolnay/rust-toolchain@nightly
     - run: cargo test --no-default-features --features "std simd_backend"
 
+  test-simd:
+    name: Test simd backend (nightly, aarch64)
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: dtolnay/rust-toolchain@nightly
+      with:
+        toolchain: aarch64-unknown-linux-gnu
+    - run: cargo test --no-default-features --target aarch64-unknown-linux-gnu --features "std simd_backend"
```

Comment:
How does this work without `cross`?

Reply:
Rust has a built-in cross-compiler. I've never really used `cross` to cross-compile myself; always had to roll my own due to circumstances.

Probably, the underlaying Ubuntu machine needs to install a gcc toolchain to actually work, but I hoped that the CI would give me some feedback already.

Reply:
A cross-compiler works for *builds*, but how does it *run* the tests?

`cross` uses QEMU

Reply:
Oh okay, I did not spot that. The new version is only with `cargo build`, not with `test`, based on the new `build-simd` CI snippet.

Reply:
I'd suggest using `cross` for this as it should be fine for testing NEON in CI.

Here are some examples (and it's probably fine to reuse the @RustCrypto `cross-install` workflow too)

https://github.com/RustCrypto/block-ciphers/blob/master/.github/workflows/aes.yml#L201-L249

Reply:
Seems like that's done in release/4.0 now, so I'll just drop my own commit.

I'd still like to point out that when I submit the armv7hl NEON work, it will work on nightly only for a while, so you might want to add armv7-nightly to the test matrix.